README.md

```diff
@@ -1,23 +1,31 @@
----
-description: >-
-  Phase2 has an approach to frontend development that is codified using the
-  Particle toolkit.
----
+# What is Particle?
 
-# Intro
+Particle is an opinionated set of tools and examples to:
 
-## Intro
-hey im editing this with important info
+1. Build an application-agnostic **design system** made of assets like Twig, Sass, JavaScript, images, etc
+2. Apply that **design system** to a locally-served Pattern Lab for rapid prototyping
+3. Apply that **design system** to other consuming apps like, Drupal, Grav, or \(soon\) Wordpress
 
-## doc todo's
+## Particle Features
 
-* folder & config structure need updated to current webpack structure
-* explain particle.js and how the particle\(\) function smashes various webpack configs together
-* explain webpack structure \(css modes, etc\)
-* troubleshooting cleanup \("nothing to see here", datetime error?, etc\) 
-* testing notes \(jest, pa11y, cypress?\)
-* vue-related notes
-* example workflow
-* flesh out the "thinking" sections
-* update the Drupal apps section \(what goes here vs Drupal under Frontend?\)
+* Drupal theme, Grav theme, Pattern Lab instance
```

Comment:
This list contains a mixed style of ending with and without ending sentences with periods. We should choose one and stick with it.